src/components/OpprettPlanPage/FyllUtPlanSteg/form/OPFormFields.tsx

```diff
@@ -0,0 +1,211 @@
+                    stund, er det viktig å ta et steg tilbake og se hvordan det
+                    faktisk har gått. Har det blitt lettere å jobbe? Fungerer
+                    tilpasningene som ønsket? Eller er det behov for mer støtte
+                    eller justeringer?
+                  </BodyLong>
+
+                  <BodyLong className="mb-2">
+                    For å sikre at planen fortsatt er relevant og nyttig for
+                    dere, anbefaler vi at dere evaluerer den innen{" "}
+                    <strong>fire uker</strong>. Dette gir nok tid til å erfare
+                    hva som fungerer – og hva som kanskje må endres. En god
+                    evaluering gir dere bedre støtte og øker sjansen for en
+                    tryggere vei tilbake til arbeid.
+                  </BodyLong>
+                </ReadMore>
+              }
+              fromDate={tomorrow}
+              toDate={oneYearFromNow}
+              isChangeDisabled={isChangeDisabled}
+              className="mb-8"
+            />
+          )}
+        </form.AppField>
+      </VStack>
+
+      <form.AppField name="harDenAnsatteMedvirket">
+        {(field) => (
+          <field.FormRadioGroup
+            label={formLabels.harDenAnsatteMedvirket.label}
+            description={
+              <ReadMore header="Hvorfor spør vi om dette?" className="mb-0">
+                Medarbeideren har rett til å være med og påvirke hvordan
+                arbeidsgiveren kan tilrettelegge ved sykefravær.
+                Arbeidsmiljøloven §4-6 sier at både du og medarbeideren skal
+                bistå til å finne løsninger, og at dere skal utarbeide
+                oppfølgingsplanen sammen. Medarbeideren skal, så langt det er
+                mulig, gi relevante opplysninger om arbeidsevnen.
+              </ReadMore>
+            }
+            options={[
+              {
+                value: "ja",
+                label: "Ja",
+              },
+              {
+                value: "nei",
+                label: "Nei",
+              },
+            ]}
+            isChangeDisabled={isChangeDisabled}
+            className="mb-4"
+          />
+        )}
+      </form.AppField>
+
+      <form.Subscribe selector={(state) => state.values.harDenAnsatteMedvirket}>
+        {(harDenAnsatteMedvirket) =>
+          harDenAnsatteMedvirket === "nei" && (
+            <form.AppField name="denAnsatteHarIkkeMedvirketBegrunnelse">
+              {(field) => (
+                <field.FormTextArea
+                  label={formLabels.denAnsatteHarIkkeMedvirketBegrunnelse.label}
+                  description={
+                    formLabels.denAnsatteHarIkkeMedvirketBegrunnelse.description
+                  }
+                  isChangeDisabled={isChangeDisabled}
+                />
+              )}
+            </form.AppField>
+          )
+        }
+      </form.Subscribe>
+    </>
+  ),
+});
+
+const tomorrow = (() => {
```

Comment:
Kan evnt flytte date utils til egen dateutil fil, for å minimere lengden på denne, og til potensiell gjenbruk i fremtiden